packages/twenty-server/src/core/billing/billing.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+
+import Stripe from 'stripe';
+
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+
+export type PriceData = Record<Stripe.Price.Recurring.Interval, Stripe.Price>;
+export enum AvailableProducts {
```

Comment:
I'm confused here a subscription is a product? Shouldn't we use subscription as a business term for a stripe subscription?